insights/specs/insights_archive.py

```diff
@@ -202,7 +202,7 @@ class InsightsArchiveSpecs(Specs):
     rpm_V_packages = first_file(["insights_commands/rpm_-V_coreutils_procps_procps-ng_shadow-utils_passwd_sudo_chrony", "insights_commands/rpm_-V_coreutils_procps_procps-ng_shadow-utils_passwd_sudo"])
     sap_hdb_version = simple_file("insights_commands/python_-m_insights.tools.cat_--no-header_sap_hdb_version")
     saphostctl_getcimobject_sapinstance = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostctrl_-function_GetCIMObject_-enuminstances_SAPInstance")
-    satellite_content_hosts_count = simple_file("insights_commands/sudo_-iu_postgres_psql_-d_foreman_-c_select_count_from_hosts")
+    satellite_content_hosts_count = simple_file("insights_commands/sudo_-iu_postgres_.usr.bin.psql_-d_foreman_-c_select_count_from_hosts")
```

Comment:
@huali027 this needs to be a `first_file` to include any previously collected data at the old location.

Reply:
@bfahr, good catch. Thanks for the review, I'll update it.c